src/test/acceptance/features/hooks.js

```diff
@@ -0,0 +1,16 @@
+const { Before, After } = require('cucumber')
+const pages = require('./pages')
+
+/**
+ * Function to (re)initialise the Selenium driver and the page objects before each Scenario.
+ */
+Before(async function () {
+  pages.initialise()
+})
+
+/**
+ * Function to quite the driver at the end of each Scenario.
```

Comment:
quite -> quit (if you're making other changes)

Reply:
Yep, I'll do that on the other branch too.

Reply:
Actually, the build seems to be not updating in GitHub again so I'll push these small typos to trigger a new build.